src/patternfly/layouts/Flex/flex.scss

```diff
@@ -160,6 +167,24 @@ $pf-l-flex--variable-map: build-variable-map("pf-l-flex--spacer", $pf-l-flex--sp
         justify-content: space-evenly;
       }
 
+      // gap
+      &.pf-m-gap#{$breakpoint-name} {
+        // row gap is on by default
+        --pf-l-flex--ColumnGap: var(--pf-l-flex--ColumnGap-base);
+
+        > * {
+          --pf-l-flex--spacer: 0;
+        }
+      }
+
+      &.pf-m-column-gap#{$breakpoint-name} {
+        --pf-l-flex--ColumnGap: var(--pf-l-flex--RowGap-base);
+
+        > * {
+          --pf-l-flex--spacer: 0;
+        }
+      }
```

Comment:
There is no `.pf-m-row-gap` - should there be? I think that might be a good way to get the default spacer when used with a breakpoint suffix, eg `.pf-m-row-gap-xs.pf-m-row-gap-on-md` would use an xs gap by default, and the default on `md` screens and larger. If the default row gap is set by default in the layout, I don't know that we need `.pf-m-row-gap` on its own.